README.md

```diff
@@ -1,5 +1,10 @@
 [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree)  ![Uptime](https://img.shields.io/endpoint?url=https%3A%2F%2Fraw.githubusercontent.com%2FEddieHubCommunity%2Fmonitoring%2Fmaster%2Fapi%2Flink-free%2Fuptime.json) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT) ![GitHub release (latest by date)](https://img.shields.io/github/v/release/EddieHubCommunity/LinkFree?style=for-the-badge)
 
+!NOTICE!
```

Comment:
What if we use a `##` or a `###` header here, so people will notice it easier?


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
I think it will look a bit weird to add a heading `##` or `###` before the main heading `# LinkFree by EddieHub`. Also, against the rules of formatting! 👀

But we can just **`bold`** it for more visibility!

Reply:
Thank you for the suggestion! I think there are more improvements that can be made 👍 

But I think this adds value already so I will merge and someone else can improve even further